src/views/Items/Item.js

```diff
@@ -27,55 +27,27 @@ export default class Item extends Component {
   };
 
   render() {
-    const description = <div className='description'>{this.props.children || this.props.description}</div>
-    const extra = <div className='extra'>{this.props.extra}</div>
-    const header = <div className='header'>{this.props.header}</div>
+    const { children, className, contentClassName, description, extra, header, image, meta, ...rest } = this.props
 
-    const imageClasses = classNames(
-      'sd-item-image',
-      'ui',
-      _.get(this.props, 'image.props.className'),
-      'image'
-    )
-    const imageProps = _.omit(_.get(this.props, 'image.props', {}), 'className')
-    const image = (
-      <div className={imageClasses}>
-        <img {...imageProps} />
-      </div>
-    )
-    const meta = <div className='meta'>{this.props.meta}</div>
+    const { className: imageClassName, ...imageProps } = _.get(image, 'props', {})
 
-    const contentClass = classNames('sd-item-content', this.props.contentClass, 'content')
-    const content = (
-      <div className={contentClass}>
-        {this.props.header && header}
-        {this.props.meta && meta}
-        {(this.props.children || this.props.description) && description}
-        {this.props.extra && extra}
-      </div>
-    )
-    const hasContent = !!this.props.header || !!this.props.meta || !!this.props.extra || !!this.props.children
-    || !!this.props.description
+    const classes = cx('sd-item', className, 'item')
+    const imageClasses = cx('sd-item-image ui', imageClassName, 'image')
+    const contentClasses = cx('sd-item-content', contentClassName, 'content')
 
-    const classes = classNames(
-      'sd-item',
-      this.props.className,
-      'item'
-    )
-    const props = _.clone(this.props)
-    // Delete all static PropTypes props in cloned porps object before spreading of props onto rendered component
-    delete props.children
-    delete props.className
-    delete props.description
-    delete props.extra
-    delete props.header
-    delete props.image
-    delete props.meta
+    const _description = children || description
+
+    const content = !header && !meta && !extra ? _description : [
```

Comment:
It may be more clear to say `(header || meta || description)` , but I see why you did it in this order for easier code layout. It's just a little confusing.